### PR TITLE
[Merged by Bors] - docs(LinearAlgebra/Matrix/Determinant/Basic): det_submatrix_equiv_self

### PR DESCRIPTION
**Pull request by madvorak**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/LinearAlgebra/Matrix/Determinant/Basic.lean [hunk 1]
```diff
@@ -209,10 +209,10 @@ theorem det_permute' (σ : Perm n) (M : Matrix n n R) :
     (M.submatrix id σ).det = Perm.sign σ * M.det := by
   rw [← det_transpose, transpose_submatrix, det_permute, det_transpose]
 
-/-- Permuting rows and columns with the same equivalence has no effect. -/
+/-- Permuting rows and columns with the same equivalence does not change the determinant. -/
 @[simp]
 theorem det_submatrix_equiv_self (e : n ≃ m) (A : Matrix m m R) :
-    det (A.submatrix e e) = det A := by
+    (A.submatrix e e).det = A.det := by
```
[the remark this answers is not in the document]
Comment (madvorak) [reply Mathlib/LinearAlgebra/Matrix/Determinant/Basic.lean:215]: Does it mean that the rest of the file should be refactored to `det ` instead?

[the remark this answers is not in the document]
Comment (Vierkantor) [reply Mathlib/LinearAlgebra/Matrix/Determinant/Basic.lean]: I don't know if that is worth the churn, especially given that `det A` vs `A.det` is more of a hunch than a style guide rule. Maybe we can open a Zulip discussion first?

[the remark this answers is not in the document]
Comment (madvorak) [reply Mathlib/LinearAlgebra/Matrix/Determinant/Basic.lean]: Let me ignore it for now and just change the comment.

**Review by Vierkantor — APPROVED** [review]
Alright, thank you!

bors r+

**Pull request closed without merging**